tach.toml

```
@@ -0,0 +1,16 @@
+exclude = [
+    ".*__pycache__",
+    ".*egg-info",
+    "docs",
+    "tests",
+]
+source_roots = [
+    ".",
+    "pennylane",
+]
+
+[[modules]]
+path = "pennylane.labs"
+depends_on = [
+    { path = "<root>" },
+]
```

Optional, but maybe also the following to be explicit?
```suggestion
depends_on = [
    { path = "<root>" },
]
visibility = []
```

---

This is the format generated directly by tach's configuration steps. I'd rather we keep this as is for now. We can always add later if it becomes needed.